massgov/mayflower
# React/dp 9399  add backstop

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)

## Description
Adds backstop tests for Mayflower/react

## Related Issue / Ticket
https://jira.mass.gov/browse/DP-9399

## Steps to Test
* `git checkout react/DP-9399--add-backstop`
* `nvm use && npm install`
* `npm run build-storybook` # the backstop tests will access the static build of the Storybook
* `npm run backstop`

* Also tests in Circle on this PR should all pass

## Screenshots
n/a

## Additional Notes:
n/a

#### Impacted Areas in Application
* added npm run commands for backstop testing in /react

#### @TODO
* add CHANGELOG
